test/parallel/test-vm-getter-throw.js

```diff
@@ -0,0 +1,20 @@
+'use strict';
+
+require('../common');
+const assert = require('assert');
+
+const vm = require('vm');
+
+const x = {};
+
+const code = `
+  var foo = 1;
+  Object.defineProperty(this, "bar", {
+    enumerable: true,
+    get: function () { throw 42 }
+  });
+  var baz = 2;
+`;
+vm.runInNewContext(code, x);
+
+assert.deepEqual(x, { foo: 1, baz: 2 });
```

Comment:
@fhinkel If I am not wrong, the intention of this patch is to skip `bar`, right?